src/fu-util.c

```diff
@@ -2393,6 +2551,60 @@ fu_util_security (FuUtilPrivate *priv, gchar **values, GError **error)
 		return FALSE;
 	str = fu_util_security_attrs_to_string (attrs);
 	g_print ("%s\n", str);
+
+	/* can we find a remote with a security attr */
+	remote = fu_util_get_remote_with_security_report_uri (priv, &error_local);
+	if (remote == NULL) {
+		g_debug ("failed to find suitable remote: %s", error_local->message);
+		return TRUE;
+	}
+
+	/* ask if we can upload them to the LVFS */
+	if (priv->no_unreported_check)
```

Comment:
I think this should go before we look for a remote that supports it

Reply:
Agree, fixed.